src/content/blog/year-in-review-2024.mdx

```diff
@@ -0,0 +1,716 @@
+---
+title: "2024 year in review"
+description: "Revisit everything that happened in Astro last year: new feature releases, survey results, Astro Together, and more!"
+publishDate: "January 16, 2025"
+authors:
+  - shinya
+  - sarah
+coverImage: "/src/content/blog/_images/year-in-review-2024/blog-post-2024-in-review.webp"
+socialImage: "/src/content/blog/_images/year-in-review-2024/og-2024-in-review.webp"
+lang: "en"
+---
+
+import Tweet from "/src/components/Tweet.astro";
+import Note from "/src/components/Note.astro";
+import Mention from "/src/components/Mention.astro";
+import YouTubeGrid from './_whats-new-components/YouTubeGrid.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import BlogContentImage from "/src/components/BlogContentImage.astro";
+import islandDiag from "/src/content/blog/_images/future-of-astro-server-islands/dark-mode-server-islands-diag.webp";
+import astroBuild from "/src/content/blog/_images/year-in-review-2024/astro-build.webp";
+import npmTrends from "/src/content/blog/_images/year-in-review-2024/npm-trends.webp";
+import starHistory from "/src/content/blog/_images/year-in-review-2024/star-history.webp";
+import stateofjs from "/src/content/blog/_images/year-in-review-2024/stateofjs.webp";
+import clack from "/src/content/blog/_images/year-in-review-2024/clack.webp";
+import bombshell from "/src/content/blog/_images/year-in-review-2024/bombshell.webp";
+
+<Note title="Editor’s Note">
+
+This is a guest post written by Shinya Fujino, an active Astro community member and contributor. This post first appeared on [Zenn.dev](https://zenn.dev/morinokami/articles/astro-2024-2025), and we would like to thank Shinya for providing this English translation to share on our blog!
```

Comment:
Maybe to avoid the double “post”?

```suggestion
This is a guest post written by Shinya Fujino, an active Astro community member and contributor. This article first appeared on [Zenn.dev](https://zenn.dev/morinokami/articles/astro-2024-2025), and we would like to thank Shinya for providing this English translation to share on our blog!
```